src/components/grid-aware/ProgramBlock/ProgramBlock.module.css

```diff
@@ -0,0 +1,139 @@
+/* Background bleed */
+
+.bleedWrapper {
+  display: grid;
+  grid-template-columns: var(--bleed-grid-template-columns);
+  margin: auto;
+  margin-bottom: 100px;
+  max-width: var(--bleed-max-width);
+}
+
+/* Override grid item default of auto so that item contents cannot stretch the grid itself */
+.bleedWrapper > * {
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .bleedWrapper {
+    grid-template-columns: 40px 1fr 40px;
+    margin-bottom: 80px;
+  }
+}
+
+.bleedMainContent {
+  grid-column: 2;
+}
+
+
+/* Main grid */
+
+.gridParent {
+  column-gap: var(--grid-column-gap);
+  display: grid;
+  grid-template-columns: repeat(var(--grid-column-count), minmax(0, 1fr));
+}
+
+.gridParent > * {
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .gridParent {
+    display: flex;
+    flex-direction: column-reverse;
+  }
+}
+
+
+/* Main grid areas */
+
+.gridAreaText {
+  grid-column: 6/ span 6;
+  grid-row: 1;
+}
+
+@media (--tablet-and-down) {
+  .gridAreaText {
+    padding-top: 16px;
+  }
+}
+
+.gridAreaImage {
+  grid-column: 2 / span 4;
+  grid-row: 1;
+}
+
+/* Left grid area */
+
+.title {
+  font: var(--font-body-large);
+  margin-bottom: 5px;
+  margin-top: 0;
+}
+
+@media (--tablet-and-down) {
+  .title {
+    margin-top: 0;
+  }
+}
+
+.subtitle {
+  font: var(--font-subtitle);
+}
+
+.description {
+  font: var(--font-body-medium);
+  margin-top: 20px;
+}
+
+.description :first-child {
+  margin: 0;
```

Comment:
Setting a margin of 0 might be too extreme here. I think you are just looking to eliminate the top margin of the first element. Since any arbitrary HTML element can appear here, you don't want to overly override styles for it.